meta/sai_serialize.h

```diff
@@ -12,13 +12,18 @@ extern "C" {
 #include <streambuf>
 #include <iomanip>
 #include <map>
+#include <unordered_map>
 #include <tuple>
 #include <cstring>
 
 #include "swss/logger.h"
 
 #include "sairedis.h"
 
+extern const std::unordered_map<sai_link_event_damping_algorithm_t, std::string> sai_link_event_damping_algorithm_to_name_map;
+
+extern const std::unordered_map<sai_redis_port_attr_t, std::string> sai_redis_port_attr_to_name_map;
+
```

Comment:
this is not needed, here, remove, and mark those in cpp files as STATIC

Reply:
we don't need to expose those to external compilation modules

Reply:
done.